### PR TITLE
add support for austrian Stuzza Namespace

### PR DESCRIPTION
**Pull request by lcx**
Why even bother defining standards when everyone goes on and creates their own standard.
Oh well, here comes support for the Austrian Stuzza Namespace (which apparentely isn't even used by every bank)

### CODE REVIEW

**Comment by tobischo** [comment]
Thank you for the contribution.

Could you summarize how the Austrian Stuzza format differs from the the standard?

Also one of the tests failed in Ruby 2.1. While it is not officially supported anymore, we still have it in a few places at the moment. Therefore we still require tests to pass also for Ruby 2.1 before merging anything.

Another option that comes to mind to solve this would be to allow registering namespaces to be mapped to a specific parser.
Then you could register the stuzza namespace without requiring us to support it in general.

**Comment by lcx** [comment]
to be hones I did not look at the differences. My bank also uses the same format and I have been using this gem for months with success.
Needed it for a client now and looks like their bank uses the other namespace which forced me to update it.
From a daily use point of view I didn't find any differences which kind of speaks for the "registering namespaces" option.

**Comment by tobischo** [comment]
I prepared something here https://github.com/Barzahlen/camt_parser/pull/18

It's waiting to be reviewed but that should solve your problem

**Comment by tobischo** [comment]
#18 has been merged and the gem released as v2.4.0
Please let us know whether that solves your problem or not.

You should be able to register the stuzza namespace for your usecase now without requiring a change to the gem

**Comment by tobischo** [comment]
@lcx Please let us know in case v2.4.0 does not work for you to register the stuzza namespace for your usecase.

I am closing this pull request for now
